### DISCUSSION TITLE
Move never combined icons of the same program seperately, not as a group

### DISCUSSION THREAD

**Discussion by Dennizzzzz** [valinet/ExplorerPatcher#1850] [Ideas]
When 'Combine taskbar icons' is set to 'Never combine', multiple icons of the same program (for example: windows explorer) are shown next to eachother, and it is not possible to move one of the icons to another place, they are stuck together as a group, and you cannot change the order.

I like to change the order of the icons of the same program. 
There is a program that had this feature that I always used on Windows 10 (its called 7+ Taskbar Tweaker), maybe you can look at this as a reference?

**Comment by pyrates999** [comment]
It's not possible because the functionality that 7+ taskbar tweaker depended on in windows 10 doesn't exist in windows 11, so it would have to be rewritten from scratch.  Here's a post on more details about it:

https://ramensoftware.com/7-taskbar-tweaker-and-a-first-look-at-windows-11

**Comment by Bunny-Head** [comment]
Is this an issue that can't be fixed somehow?
